packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -31,7 +31,7 @@
         :key="imageSrc"
         :alt="result.name"
         :src="imageSrc"
-        class="x-picture__image x-result-picture__image"
+        class="x-result-picture__image"
```

Comment:
I think this class can be renamed to `x-result-picture-image`.
Update the selector below too. (Check if it can affect something, just in case)